cmd/serverless/dependencies_linux_amd64.txt

```diff
@@ -159,6 +159,7 @@ github.com/DataDog/datadog-agent/pkg/logs/internal/framer
 github.com/DataDog/datadog-agent/pkg/logs/internal/parsers
 github.com/DataDog/datadog-agent/pkg/logs/internal/parsers/dockerfile
 github.com/DataDog/datadog-agent/pkg/logs/internal/parsers/encodedtext
+github.com/DataDog/datadog-agent/pkg/logs/internal/parsers/integrations
```

Comment:
Not a blocker, but something to have in mind. It would be interesting to see if we could make the different parsers configurable in the Logs package. It seems odd the serverless binary includes them as there are no integrations running in that environment